### ISSUE TITLE
A11Y - Filter & Find actions not accessible using keyboard shortcuts

### ISSUE THREAD

**Issue by jisedlac** [oracle/visualvm#54]
Filter and Find actions in various views cannot be invoked using keyboard shortcuts. In the NetBeans profiler this works correctly.

Affected views - Sampler/Profiler live results/snapshots, Threads, Heap Viewer.
